.github/workflows/ci.yaml

```
@@ -312,6 +312,25 @@ jobs:
                   cmakeAppendedArgs: "-DCMAKE_BUILD_TYPE=Release -DSLINT_FEATURE_INTERPRETER=1 -DSLINT_FEATURE_BACKEND_QT=1"
                   buildDirectory: ${{ runner.workspace }}/examples/build
                   buildWithCMakeArgs: "--config Release"
+    
+    vsce_build_test:
+        strategy:
+            matrix:
+                os: [ubuntu-22.04]
+
+        runs-on: ${{ matrix.os }}
+
+        steps:
+            - uses: actions/checkout@v4
+            - name: Run npm install
```

Suggestion to add two steps:

1) Linux dependencies so that the slint-lsp build will hopefully pass in the same way that we build it elsewhere
2) Use the same node version as we use for the production build in nightly_snapshot.yaml

```suggestion
            - uses: actions/checkout@v4
            - uses: ./.github/actions/install-linux-dependencies            
            - uses: actions/setup-node@v4
              with:
                  node-version: 18            
            - name: Run npm install
```